src/components/Pagination.jsx

```diff
@@ -84,7 +97,7 @@ function Pagination({
           // Do not remove the aria-label below, it is used for Hatchways automation.
           aria-label="Goto next page"
           onClick={onNext}
-          disabled={false} // change this line to disable a button.
+          disabled={currentPage === paginationRange.slice(-1)[0]} // disables next on the last page
```

Comment:
It would make for better memory space efficiency to avoid making unnecessary shallow copies of the "paginationRange" array. You could access the last page in the paginationRange.

Reply:
Good point, will correct.